update.php

```diff
@@ -1210,114 +1223,153 @@ function update_1136() {
-	if(! $r)
-		return UPDATE_FAILED ;
-	return UPDATE_SUCCESS ;
+	$r = q("ALTER TABLE `profile` ADD `howlong` DATETIME NOT NULL DEFAULT '0000-00-00 00:00:00' AFTER `with`");
+
+	if ($r) {
+		return UPDATE_SUCCESS ;
+	}
+
+	return UPDATE_FAILED ;
 }
 
 function update_1146() {
-	$r = q("alter table profile add hometown char(255) not null after `country-name`, add index ( `hometown` ) ");
-	if(! $r)
-		return UPDATE_FAILED ;
-	return UPDATE_SUCCESS ;
+	$r = q("ALTER TABLE `profile` ADD `hometown` CHAR(255) NOT NULL AFTER `country-name`, ADD INDEX ( `hometown` ) ");
+
+	if ($r) {
+		return UPDATE_SUCCESS ;
+	}
+
+	return UPDATE_FAILED ;
 }
 
 function update_1147() {
 	$r1 = q("ALTER TABLE `sign` ALTER `iid` SET DEFAULT '0'");
 	$r2 = q("ALTER TABLE `sign` ADD `retract_iid` INT(10) UNSIGNED NOT NULL DEFAULT '0' AFTER `iid`");
 	$r3 = q("ALTER TABLE `sign` ADD INDEX ( `retract_iid` )");
-	if((! $r1) || (! $r2) || (! $r3))
-		return UPDATE_FAILED ;
-	return UPDATE_SUCCESS ;
+
+	if ($r1 && $r2 && $r3) {
+		return UPDATE_SUCCESS ;
+	}
+
+	return UPDATE_FAILED ;
 }
 
 function update_1148() {
-	$r = q("ALTER TABLE photo ADD type CHAR(128) NOT NULL DEFAULT 'image/jpeg' AFTER filename");
-	if (!$r)
-		return UPDATE_FAILED;
-	return UPDATE_SUCCESS;
+	$r = q("ALTER TABLE `photo` ADD `type` CHAR(128) NOT NULL DEFAULT 'image/jpeg' AFTER `filename`");
+
+	if ($r) {
+		return UPDATE_SUCCESS ;
+	}
+
+	return UPDATE_FAILED ;
 }
 
 
 function update_1149() {
-	$r1 = q("ALTER TABLE profile ADD likes text NOT NULL after prv_keywords");
-	$r2 = q("ALTER TABLE profile ADD dislikes text NOT NULL after likes");
-	if (! ($r1 && $r2))
-		return UPDATE_FAILED;
-	return UPDATE_SUCCESS;
+	$r1 = q("ALTER TABLE `profile` ADD `likes` TEXT NOT NULL AFTER `prv_keywords`");
+	$r2 = q("ALTER TABLE `profile` ADD `dislikes` TEXT NOT NULL AFTER `likes`");
+
+	if ($r1 && $r2) {
+		return UPDATE_SUCCESS;
+	}
+
+	return UPDATE_FAILED;
 }
 
 
 function update_1150() {
-	$r = q("ALTER TABLE event ADD summary text NOT NULL after finish, add index ( uid ), add index ( cid ), add index ( uri ), add index ( `start` ), add index ( finish ), add index ( `type` ), add index ( adjust ) ");
-	if(! $r)
-		return UPDATE_FAILED;
-	return UPDATE_SUCCESS;
+	$r = q("ALTER TABLE `event` ADD `summary` TEXT NOT NULL AFTER `finish`, ADD INDEX ( `uid` ), ADD INDEX ( `cid` ), ADD INDEX ( `uri` ), ADD INDEX ( `start` ), ADD INDEX ( `finish` ), ADD INDEX ( `type` ), ADD INDEX ( `adjust` ) ");
+
+	if ($r) {
+		return UPDATE_SUCCESS ;
+	}
+
+	return UPDATE_FAILED ;
 }
 
 
 function update_1151() {
-	$r = q("CREATE TABLE IF NOT EXISTS locks (
-			id INT NOT NULL AUTO_INCREMENT PRIMARY KEY ,
-			name CHAR( 128 ) NOT NULL ,
-			locked TINYINT( 1 ) NOT NULL DEFAULT '0'
+	$r = q("CREATE TABLE IF NOT EXISTS `locks` (
+			`id` INT NOT NULL AUTO_INCREMENT PRIMARY KEY ,
+			`name` CHAR( 128 ) NOT NULL ,
+			`locked` TINYINT( 1 ) NOT NULL DEFAULT '0'
 		  ) ENGINE = MYISAM DEFAULT CHARSET=utf8 ");
```

Comment:
We should remove the "ENGINE = MYISAM" from the update since we now prefer using InnoDB.